docs/hub/local-apps.md

```diff
@@ -0,0 +1,93 @@
+# Use AI Models Locally
+
+You can run AI models from the Hub locally on your machine. This means that you can benefit from these advantages:
+
+- **Privacy**: You won't be sending your data to a remote server.
+- **Speed**: Your hardware is the limiting factor, not the server or connection speed.
+- **Control**: You can configure models to your liking.
+- **Cost**: You can run models locally without paying for an API provider.
+
+## How to Use Local Apps
+
+Local apps are applications that can run Hugging Face models directly on your machine. To get started:
+
+1. **Enable local apps** in your [Local Apps settings](https://huggingface.co/settings/local-apps).
+
+![Local Apps](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/local-apps/settings.png)
+
+1. **Choose a supported model** from the Hub by searching for it.
+
+![Local Apps](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/local-apps/search_llamacpp.png)
+
+3. **Select the local app** from the "Use this model" dropdown on the model page.
+
+![Local Apps](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/local-apps/button.png)
+
+4. **Copy and run** the provided command in your terminal.
+
+![Local Apps](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/local-apps/command.png)
+
+## Supported Local Apps
+
+The best way to check if a local app is supported is to go to the Local Apps settings and see if the app is listed. Here is a quick overview of some of the most popular local apps:
```

Comment:
Maybe we could add a link for devs that want to add new local apps?

Reply:
don't think it's necessary, since we add them more opportunistically atm.